Content.Server/GameTicking/Rules/ZombieRuleSystem.cs

```diff
@@ -130,6 +133,23 @@ private void CheckRoundEnd(ZombieRuleComponent zombieRuleComponent)
             _roundEnd.EndRound();
     }
 
+    /// <summary>
+    ///     Doesn't let to recall the shuttle. No roundstalling
```

Comment:
Just a quick doc comment change

```suggestion
    ///     Doesn't let the shuttle be recalled, stopping round-stalling.
```